provider/postgis/util.go

```diff
@@ -16,8 +16,13 @@ import (
 	"github.com/jackc/pgx/pgtype"
 )
 
+// isMVT will return true if the provider is MVT based
+func isMVT(providerType string) bool {
+	return providerType == "mvt_postgis"
```

Comment:
minor nit, but since we're using `"mvt_postgis"` in more than 1 place (register.go and here), we should probably move it to a const.